### PR TITLE
14.0 [MIG] base_module_record: Migrated base_module_record Module.

### PR DESCRIPTION
**Pull request by Rajan-SerpentCS**

### CODE REVIEW

File: base_module_record/LICENSE/LICENSE [hunk 1]
```diff
@@ -0,0 +1,16 @@
+    OpenERP, Open Source Management Solution
+    Copyright (C) 2004-2010 OpenERP SA (<http://www.openerp.com>)
+    Copyright (C) 2011-Today Serpent Consulting Services Pvt. Ltd. (<http://www.serpentcs.com>).
```
Comment (JayVora-SerpentCS) [inline base_module_record/LICENSE/LICENSE]: 2021-Today

File: base_module_record/i18n/ar.po [hunk 2]
```diff
@@ -0,0 +1,267 @@
+# Translation of OpenERP Server.
```
Comment (JayVora-SerpentCS) [inline base_module_record/i18n/ar.po]: can you regenerate ALL the PO files please?

File: base_module_record/LICENSE/LICENSE (hunk 1 above)
Comment (Rajan-SerpentCS) [reply base_module_record/LICENSE/LICENSE]: Done

File: base_module_record/i18n/ar.po (hunk 2 above)
Comment (Rajan-SerpentCS) [reply base_module_record/i18n/ar.po]: Done

**Pull request merged**